### PR TITLE
OSDOCS-1977: Adding a 'Preparing to install on OpenStack' assembly

### PR DESCRIPTION
**Pull request by pneedle-rh**
This applies to master and branch/enterprise-4.8.

This PR relates to https://issues.redhat.com/browse/OSDOCS-1977. The PR adds a "Preparing to install on OpenStack" assembly that provides high level guidance about the different installation methods available to install OpenShift on Red Hat OpenStack Platform. I have also added one small edit to the existing "Preparing to install on AWS" assembly in this PR.

The preview is [here](https://deploy-preview-33455--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/preparing-to-install-on-openstack.html).

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Thanks!

File: installing/installing_openstack/preparing-to-install-on-openstack.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+* **xref:../../installing/installing_openstack/installing-openstack-installer-kuryr.adoc#installing-openstack-installer-kuryr[Installing a cluster on OpenStack with Kuryr]**: You can install a customized {product-title} cluster on {rh-openstack} that uses Kuryr SDN. Kuryr and {product-title} integration is primarily designed for {product-title} clusters running on {rh-openstack} VMs. Kuryr improves the network performance by plugging {product-title} pods into {rh-openstack} SDN. In addition, it provides interconnectivity between pods and {rh-openstack} virtual instances.
+
+* **xref:../../installing/installing_openstack/installing-openstack-user.adoc#installing-openstack-user[Installing a cluster on OpenStack on your own infrastructure]**: You can install {product-title} on {rh-openstack} that runs on user-provisioned infrastructure. By using this installation method, you can integrate your cluster with existing infrastructure and modifications. For user-provisioned installations, you must create all {rh-openstack} resources, like Nova servers, Neutron ports, and security groups. You can use the provided Ansible playbooks to assist with the deployment process.
```
Comment (mikemckiernan) [inline installing/installing_openstack/preparing-to-install-on-openstack.adoc:24]: Nit: "user-provisioned installations", seems like a new term.  Could it be "For installations on user-provisioned infrastructure, you must..."

Or, it really is a term that is new to me.

File: installing/installing_openstack/preparing-to-install-on-openstack.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+* **xref:../../installing/installing_openstack/installing-openstack-user-sr-iov.adoc#installing-openstack-user-sr-iov[Installing a cluster on OpenStack on your own SR-IOV infrastructure]**: You can install {product-title} on {rh-openstack} that runs on user-provisioned infrastructure and uses single-root input/output virtualization (SR-IOV) networks to run compute machines.
+
+* **xref:../../installing/installing_openstack/installing-openstack-installer-restricted.adoc#installing-openstack-installer-restricted[Installing a cluster on OpenStack in a restricted network]**: You can install {product-title} on {rh-openstack} in a restricted or disconnected network, by creating an internal mirror of the installation release content. You can use this method to install a cluster that does not require an active Internet connection to obtain the software components. You can also use this installation method to ensure that your clusters only use container images that satisfy your organizational controls on external content.
```
Comment (mikemckiernan) [inline installing/installing_openstack/preparing-to-install-on-openstack.adoc:30]: I believe internet can be lowercase, ISG p348.

**Review by mikemckiernan — APPROVED** [review]
looks good

File: installing/installing_openstack/preparing-to-install-on-openstack.adoc (hunk 2 above)
Comment (pneedle-rh) [reply installing/installing_openstack/preparing-to-install-on-openstack.adoc:30]: Thanks for highlighting this. Resolved.

**Pull request merged**